.github/workflows/R-CMD-check.yaml

```diff
@@ -110,3 +116,19 @@ jobs:
         with:
           name: ${{ runner.os }}-r${{ matrix.config.r }}-results
           path: check
+
+  check-orderly-server:
+
+    runs-on: ubuntu-20.04
+
+    needs: R-CMD-check
+
+    steps:
+      - uses: convictional/trigger-workflow-and-wait@v1.6.1
```

Comment:
Managed to find an existing step which will trigger it and wait. See https://github.com/marketplace/actions/trigger-workflow-and-wait